interface/views.py

```diff
@@ -327,3 +328,21 @@ def reveal(request, course_pk, assignment_pk):
         'HTTP_REFERER',
         f'/assignment/{course.pk}/{assignment.pk}',
     ))
+
+
+@login_required
+@staff_member_required
+def code_view(request, pk, filename):
+    submission = get_object_or_404(Submission, pk=pk)
+
+    if (request.user
+            not in submission.assignment.course.teaching_assistants.all()):
```

Comment:
Q: Might be worth creating a variable for this:
Ex:
```all_tas = submission.assignment.course.teaching_assistants.all()```
and then check:
```
if request_user not in all_tas:
```
Such that we do not split the if statement in two lines